js/books.js

```diff
@@ -0,0 +1,116 @@
+function localStorageAvailable() {
+  let storage;
+
+  try {
+    storage = window.localStorage;
+    const x = '__storage_test__';
+    storage.setItem(x, x);
+    storage.removeItem(x);
+    return true;
+  } catch (e) {
+    return e instanceof DOMException && (
+      // everything except Firefox
+      e.code === 22
+      // Firefox
+      || e.code === 1014
+      // test name field too, because code might not be present
+      // everything except Firefox
+      || e.name === 'QuotaExceededError'
+      // Firefox
+      || e.name === 'NS_ERROR_DOM_QUOTA_REACHED')
+      // acknowledge QuotaExceededError only if there's something already stored
+      && (storage && storage.length !== 0);
+  }
+}
+
+function setItem(name, item) {
+  if (localStorageAvailable()) {
+    localStorage.setItem(name, item);
+  }
+}
+
+function getItem(name) {
+  if (localStorageAvailable()) {
+    return localStorage.getItem(name);
+  }
+
+  return null;
+}
+
+/* eslint-disable no-use-before-define */
+function saveBooks() {
+  setItem('books', JSON.stringify(books));
+}
+
+function getBooks() {
+  return JSON.parse(getItem('books'));
+}
+
+function displayBooks() {
+  const booksDiv = document.querySelector('.books');
+  booksDiv.textContent = '';
+
+  const createDiv = () => document.createElement('div');
+
+  books.forEach((book) => {
+    const bookDiv = createDiv();
+    bookDiv.className = 'book';
+
+    const titleDiv = createDiv();
+    titleDiv.textContent = book.title;
+
+    const authorDiv = createDiv();
+    authorDiv.textContent = book.author;
+
+    const removeBtn = document.createElement('button');
+    removeBtn.className = ['removeBtn-', book.id].join('');
+    removeBtn.textContent = 'Remove';
+    removeBtn.addEventListener('click', (event) => {
+      const source = event.target;
+      const bookId = source.className.split('-')[1];
+
+      removeBook(bookId);
+    });
```

Comment:
-[OPTIONAL]  prefer the use of `DOMContentLoaded` over `load` "A different event, load, should be used only to detect a fully-loaded page. It is a common mistake to use load where DOMContentLoaded would be more appropriate.", [see](https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event)